spring-boot-project/spring-boot-docs/src/docs/asciidoc/production-ready-features.adoc

```diff
@@ -1377,6 +1377,7 @@ Spring Boot Actuator provides dependency management and auto-configuration for h
 - <<production-ready-metrics-export-graphite,Graphite>>
 - <<production-ready-metrics-export-humio,Humio>>
 - <<production-ready-metrics-export-influx,Influx>>
+- <<production-ready-metrics-export-instana,Instana>>
```

Comment:
This doesn't feel quite right to me. This is a list of monitoring systems that Micrometer supports and Micrometer itself does not support Instana.